finrl/meta/paper_trading/paper_trader.py

```
@@ -19,9 +19,9 @@
 from finrl.meta.data_processors.processor_alpaca import AlpacaProcessor
 from finrl.meta.paper_trading.alpaca import PaperTradingAlpaca
 from finrl.meta.paper_trading.futu import PaperTradingFutu
-
 from elegantrl.agents import *
 
+MODELS = {"ddpg": AgentDDPG, "td3": AgentTD3, "sac": AgentSAC, "ppo": AgentPPO, "a2c": AgentA2C}
```

Ruff

`AgentDDPG` may be undefined, or defined from star imports

(F405)